sale_order_color_code_tintometric_system_stock/models/stock_move.py

```diff
@@ -0,0 +1,15 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+    
+    color_code_tintometric = fields.Char(compute="_compute_color_code_tintometric", string="Lot/S.Tin")
+
+    @api.depends('picking_id.sale_id.order_line.color_code_tintometric')
+    def _compute_color_code_tintometric(self):
+        for move in self:
+            if move.sale_line_id:
+                move.color_code_tintometric = move.sale_line_id.color_code_tintometric
```

Comment:
Much simpler (since there's a direct relation for this case):
```suggestion
    color_code_tintometric = fields.Char(related="sale_line_id.color_code_tintometric", string="Lot/S.Tin")
```